index.js

```diff
@@ -26,9 +26,16 @@ module.exports = function(options) {
         runOptions.push(file);
     });
 
-    runOptions.push('-f');
-    var format = options.format || 'pretty';
-    runOptions.push(format);
+    if (!options.format) {
+        options.format = 'pretty';
+    }
+
+    var formats = util.isArray(options.format) ? options.format : [options.format];
```

Comment:
Seems like `util.isArray` is deprecated and should be replaced to `Array.isArray` now. https://nodejs.org/api/util.html#util_util_isarray_object